task3/task_3_A/main_3_A.js

```diff
@@ -0,0 +1,78 @@
+const DOWNLOADS = [
+  {
+    id: 1,
+    title: 'Recipe',
+    status: 'Done',
+  },
+  {
+    id: 2,
+    title: 'Workouts',
+    status: 'Pending',
+  },
+  {
+    id: 3,
+    title: 'Passwords',
+    status: 'Pending',
+  },
+  {
+    id: 4,
+    title: 'To Do 2021',
+    status: 'Pending',
+  },
+  {
+    id: 5,
+    title: 'Books',
+    status: 'Failed',
+  },
+];
+
+const table = document.querySelector('.table');
+const button = document.querySelector('.button');
+const pending = 'Pending';
+const done = 'Done';
```

Comment:
It will be better to create such variable for couple of statuses:
```
const DOWNLOAD_STATUSES = {
  done: "Done",
  pending: "Pending",
  failed: "Failed",
}
```